src/App.jsx

```diff
@@ -14,70 +15,92 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [goodState, setGood] = useState('Jam');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  return (
+    <main className="section container">
+      <h1 className="title is-flex is-align-items-center">
+        <span>
+          {!goodState ? 'No goods selected' : `${goodState} is selected`}
+        </span>
 
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
-
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
+        {!goodState
+          ? ''
+          : (
```

Comment:
```suggestion
        {goodState
         && (
```
you can simplify it

Reply:
Hi! Thank you for feedback! I changed that)